app/src/main/kotlin/net/syncthing/lite/utils/MimeType.kt

```diff
@@ -11,7 +14,19 @@ object MimeType {
         return mimeType ?: DEFAULT_MIME_TYPE
     }
 
-    fun getFromUrl(url: String) = getFromExtension(
-            MimeTypeMap.getFileExtensionFromUrl(url)
+    fun getFromUrl(url: String, convertExtensionToLowerCase: Boolean): String {
+        val extension = MimeTypeMap.getFileExtensionFromUrl(url)
+
+        return getFromExtension(
+                if (convertExtensionToLowerCase)
+                    extension.toLowerCase(Locale.US)
```

Comment:
You might want to use the actual locale here, or at least make sure nothing unexpected happens when non-ascii characters are used. Check this site for examples:

https://wiki.collectionspace.org/display/collectionspace/Special+Characters+and+Formatting+-+QA+Test+Plan

Reply:
The specified locale should ensure that the conversion to lower case does not differ if the user uses an other locale. File extensions only contain ASCI letter and toLowerCase ignores symbols (like numbers) which don't belong to the locale (as far as I know).

Reply:
It is always possible that someone creates a file extension that is not ascii. In that case, it might cause some unexpected behaviour that the function here uses Locale.US which would be hard to figure out. And other locales should have the same behaviour for toLowerCase() for ascii characters, right?

Reply:
Yes